modele-social/règles/entreprise-établissement.yaml

```
@@ -344,11 +344,11 @@ entreprise . imposition . IR . information sur le report de déficit:
 entreprise . exercice: oui
 entreprise . exercice . début:
   type: date
-  par défaut: 01/01/2020
+  par défaut: 01/01/2021
```

C'est possible mais un peu hacky : 
```suggestion
  par défaut: 
    variations:
      - si: durée depuis début année > 7 mois
        alors: période . début d'année n-1
      - sinon: période . début d'année
période . début d'année n-1: 01/01/2020
période . durée depuis début d'année
  durée:
    depuis: début d'année
  unité: mois
```